MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/ImageSourceTimeLapseActivity.java

```diff
@@ -0,0 +1,137 @@
+package com.mapbox.mapboxandroiddemo.examples.styles;
+
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.v7.app.AppCompatActivity;
+
+import com.mapbox.mapboxandroiddemo.R;
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.geometry.LatLng;
+import com.mapbox.mapboxsdk.geometry.LatLngQuad;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.style.layers.RasterLayer;
+import com.mapbox.mapboxsdk.style.sources.ImageSource;
+
+/**
+ * Use a series of images to create an animation with an ImageSource
+ */
+public class ImageSourceTimeLapseActivity extends AppCompatActivity implements OnMapReadyCallback {
+
+  private MapView mapView;
+  private Handler handler;
+  private Runnable runnable;
+  private static final String ID_IMAGE_SOURCE = "animated_image_source";
+  private static final String ID_IMAGE_LAYER = "animated_image_layer";
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    // Mapbox access token is configured here. This needs to be called either in your application
+    // object or in the same activity which contains the mapview.
+    Mapbox.getInstance(this, getString(R.string.access_token));
+
+    // This contains the MapView in XML and needs to be called after the access token is configured.
+    setContentView(R.layout.activity_image_source_time_lapse);
+
+    mapView = (MapView) findViewById(R.id.mapView);
+    mapView.onCreate(savedInstanceState);
+    mapView.getMapAsync(this);
+  }
+
+  @Override
+  public void onMapReady(MapboxMap mapboxMap) {
+    // Add source
+    LatLngQuad quad = new LatLngQuad(
+      new LatLng(46.437, -80.425),
+      new LatLng(46.437, -71.516),
+      new LatLng(37.936, -71.516),
+      new LatLng(37.936, -80.425));
+    mapboxMap.addSource(new ImageSource(ID_IMAGE_SOURCE, quad, R.drawable.southeast_radar_0));
+
+    // Add layer
+    RasterLayer layer = new RasterLayer(ID_IMAGE_LAYER, ID_IMAGE_SOURCE);
+    mapboxMap.addLayer(layer);
+
+    // Loop the GeoJSON refreshing
+    handler = new Handler();
+    runnable = new RefreshImageRunnable(mapboxMap, handler);
+    handler.postDelayed(runnable, 100);
+  }
+
+  private static class RefreshImageRunnable implements Runnable {
+    private MapboxMap mapboxMap;
```

Comment:
you can set this and the handler to final since they are initialized in the constructor

Reply:
Fixed.